.swiftlint.yml

```diff
@@ -21,11 +21,9 @@ disabled_rules:
   - explicit_top_level_acl
   - explicit_type_interface
   - file_types_order
-  - final_test_case
```

Comment:
There are actually many more violations. The reason for them being missed is that the rule doesn't know about `SwiftLintTestCase` being a test class. This can be configured.

Reply:
318 violations, when I configure the rule to pay attention to `SwiftLintTestCase` - I was really only intending to do a few small changes here, although I guess patching those would be pretty safe - do you think I should patch them all  now?

Reply:
Expected. 😅 236 of them are generated. Patching them would be fine.

Reply:
So a lot of those were in GeneratedTests - I've patched the stencil and regenerated, and the other non-generated tests as well.